docs/content/en/docs/workshops/introduction/harbor.md

```diff
@@ -0,0 +1,214 @@
+
+1. Login
+
+    Log in Harbor web portal with the default credential as shown below
+    ```bash
+    admin
+    Harbor12345
+    ```
+ 
+    ![Harbor web portal](/images/harbor-portal.png)
+
+1. Create a nonproxy project
+
+   ![Harbor nonproxy project](/images/harbor-nonproxy_project.png) 
+
+1. Create a registry proxy
+
+    In order for Harbor to proxy a remote private ECR registry, an IAM credential with necessary permissions need to be created. Usually, it follows three steps: 
+
+    1. Policy
+
+        This is where you specify all necessary permissions and please refer to [private repository policies](https://docs.aws.amazon.com/AmazonECR/latest/userguide/repository-policies.html), [IAM permissions for pushing an image](https://docs.aws.amazon.com/AmazonECR/latest/userguide/image-push.html) and [ECR policy examples](https://docs.aws.amazon.com/AmazonECR/latest/userguide/security_iam_id-based-policy-examples.html) to figure out the minimal set of required permissions.
+
+        For simplicity, the build-in policy AdministratorAccess is used here.
+
+        ![Harbor private ecr policy](/images/harbor-private_ecr_policy.png)
+
+    1. User group
+
+        This is an easy way to manage a pool of users who share the same set of permissions by attaching the policy to the group.
+
+        ![Harbor private ecr user group](/images/harbor-private_ecr_user_group.png)
+
+    1. User
+
+        Create a user and add it to the user group. In addition, please navigate to Security credentials to generate an access key. Access keys consists of two parts: an access key ID and a secret access key. Please save both as they are used in the next step. 
+
+        ![Harbor private ecr user](/images/harbor-private_ecr_user.png)
+
+    Navigate to `Registries` on the left panel, and then click on `NEW ENDPOINT` button. Choose `Aws ECR` as Provider, and enter `private-ecr` as Name, `https://[ACCOUNT NUMBER].dkr.ecr.us-west-2.amazonaws.com/` as Endpoint URL, use the access key ID part of the generated access key as Access ID, and use the secret access key part of the generated access key as Access Secret. Save it by click on OK.
+
+    ![Harbor private ecr proxy](/images/harbor-private_ecr_proxy.png)
+
+1. Create a replication rule
+
+    ![Harbor replication rule](/images/harbor-replication_rule.png)
+
+1. Prepare an image
+   {{% alert title="Note" color="primary" %}}
+
+   * `harbor.eksa.demo:30003` should be replaced with whatever `externalURL` is set to in the Harbor package yaml file.
+
+   {{% /alert %}}
+    ```bash
+    docker pull alpine
+    docker tag alpine:latest harbor.eksa.demo:30003/nonproxy-project/alpine:latest
+    ```
+
+1. Authenticate with Harbor with the default credential as shown below
+    ```bash
+    admin
+    Harbor12345
+    ```
+   {{% alert title="Note" color="primary" %}}
+
+   * `harbor.eksa.demo:30003` should be replaced with whatever `externalURL` is set to in the Harbor package yaml file.
```

Comment:
```suggestion
   * `harbor.eksa.demo:30003` should be replaced with whatever `externalURL` is set to in the Harbor package YAML file.
```
This is a little detail, but YAML is an acronym, so anywhere in the document that you're not using YAML as part of a filename, it should be capitalized. I mean, people are gonna know what you mean either way, but it looks more professional/better capitalized.